docs/guides/deployment/digitalocean.rst

```diff
@@ -60,27 +60,45 @@ Use these variables to change the password for the default role ``edgedb``.
          "alter role edgedb set password := '${PASSWORD}'"
    OK: ALTER ROLE
 
-You can now link and connect to the new EdgeDB instance:
+Construct the DSN
+-----------------
+
+Let's construct your instance's DSN (also known as a "connection string").
+We'll write the value to a file called ``dsn.txt`` so it doesn't get stored in
+shell logs.
 
 .. code-block:: bash
 
-   $ printf $PASSWORD | edgedb instance link \
-         --password-from-stdin \
-         --trust-tls-cert \
-         --host $IP \
-         --non-interactive \
-         digitalocean
-   Authenticating to edgedb://edgedb@your-droplet-ip:5656/edgedb
-   Trusting unknown server certificate
+   $ echo edgedb://edgedb:$PASSWORD@$IP > dsn.txt
+
+Copy the value from ``dsn.txt``. Tun the following command to open a REPL
+to the new instance.
```

Comment:
```suggestion
instance to the newly established DSN.
```

Reply:
Hm, you're not connecting to the DSN, you're connecting to the instance the DSN refers to. I clarified the wording a bit.